vision_agent/tools/tools.py

```diff
@@ -2162,6 +2210,7 @@ def _plot_counting(
     generate_pose_image,
     closest_mask_distance,
     closest_box_distance,
+    video_temporal_localization,
```

Comment:
Just remove this line, this is what adds it to the VisionAgent.

Reply:
Done!